chapters/scoping.tex

```diff
@@ -98,7 +98,12 @@ \subsection{Composite Name Lookup}\label{composite-name-lookup}
 \item
   If the identifier denotes a class, that class is temporarily flattened (as if instantiating a component without modifiers of this class, see \cref{modification-environment}) and using the enclosing classes of the denoted class.
   The rest of the name (e.g., \lstinline!B! or \lstinline!B.C!) is looked up among the declared named elements of the temporary flattened class.
-  If the class does not satisfy the requirements for a package, the lookup is restricted to encapsulated\indexinline{encapsulated} elements only.
+  The lookup will only find the element (assuming it exists) in the following cases:
+  \begin{itemize}
+  \item If the class is declared as \lstinline!package! or \lstinline!operator! (see below) all elements can be found.
```

Comment:
Clarify meaning of `operator` (removing "see below" to avoid too much parenthesized content in one place):
```suggestion
  \item If the class is declared as \lstinline!package! or \lstinline!operator! (but not \lstinline!operator record! or \lstinline!operator function!) all elements can be found.
```